cstrml/payment/src/lib.rs

```diff
@@ -118,6 +108,10 @@ decl_storage! {
         /// A mapping from storage order id to payment ledger info
         pub Payments get(fn payments):
         map hasher(twox_64_concat) T::Hash => Option<Ledger<BalanceOf<T>>>;
```

Comment:
Since we have `StakingLedger`, we can rename back to `PaymentLedger` to help `apps` and `sdk` distinguish.

Reply:
Will rename them all to PaymentLedger/PaymentLedgers